Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -1267,6 +1267,45 @@ def init_filter_args_options():
             'False': False,
         }),
         'Identity_type': AlertFilterArg('Identity_type', 'EQ', dropdown),
+        'alert_action_status': AlertFilterArg('alert_action_status', 'EQ', dropdown, {
+            'detected': 'DETECTED',
```

Comment:
Why not do something of the sort: 
```python
{v: k for k, v in ALERT_STATUS_TYPES.items()}
```

??

Reply:
We can.